pkgs/applications/terminal-emulators/st/lukesmithxyz-st.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fontconfig
+, harfbuzz
+, libX11
+, libXext
+, libXft
+, ncurses
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lukesmithxyz-st";
+  version = "0.0.0+unstable=2021-08-10";
+
+  src = fetchFromGitHub {
+    owner = "LukeSmithxyz";
+    repo = "st";
+    rev = "e053bd6036331cc7d14f155614aebc20f5371d3a";
+    hash = "sha256-WwjuNxWoeR/ppJxJgqD20kzrn1kIfgDarkTOedX/W4k=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+  buildInputs = [
+    fontconfig
+    harfbuzz
+    libX11
+    libXext
+    libXft
+    ncurses
+  ];
+
+  postPatch = ''
+    sed -i 's|.*git submodule.*||' Makefile
```

Comment:
Please apply a patch, not just run an `sed` command. These tend to rot over time.